modules/custom/d_update/src/Updater.php

```diff
@@ -137,32 +149,44 @@ public function checklist() {
   }
 
   /**
-   * Import a config file.
+   * Import a config file if the module exists.
+   *
+   * The method tries to read config files from the modules' 'install' or 'optional' directories,
+   * if the config has been found and the module exists - the config is imported.
    *
    * @param string $source
    *   Module/theme name.
    * @param string $name
    *   Config file name without .yml extension.
    * @param string $hash
    *   Hashed array with config data.
-   * @param bool $optional
-   *   Specify if config should be searched only in 'config/optional'.
    *
    * @return bool
-   *   Returns if config was imported successfully.
+   *   TRUE if the config was imported successfully or the module does not exist,
+   *   FALSE otherwise.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
    */
   public function importConfig($source, $name, $hash) {
     $data = $this->readConfigFromFile($source, $name, 'install');
+
     if (empty($data)) {
       $data = $this->readConfigFromFile($source, $name, 'optional');
     }
+
     if (empty($data)) {
       $this->logger
         ->error('Cannot find file for %config', ['%config' => $name]);
 
       return FALSE;
     }
 
+    // No need to import anything if the module does not exist.
+    if ($this->moduleHandler->moduleExists($source) === FALSE) {
+      return TRUE;
```

Comment:
Rozumiem, że w ten sposób ignorujemy configi do wyłączonych modułów? :)

Reply:
Ignorujemy, ale zwracamy state taki sam jak przy poprawnym imporcie, nie wiem do końca czy to ok

Reply:
można dodać info loggerem, podobnie jak powyzej... ale jak dla mnie jest ok.

Reply:
@jakub-wozniak-droptica czy to moduleExists nie działa tak podobnie do `$this->moduleExtensionList->getList();` który jest już w `Updater.php` ?
![image](https://user-images.githubusercontent.com/37533838/86885036-d9643e00-c0f4-11ea-9e4e-9c796d0caf0e.png)

Reply:
lukne i jak tak to wrzuce poprawke